shielddash/api/models.py

```diff
@@ -1,8 +1,36 @@
 from django.db import models
+from django.utils import timezone
 
 
-class Retention(models.Model):
+class Study(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(default='')
+    start_time = models.DateTimeField(blank=True, null=True)
+    end_time = models.DateTimeField(blank=True, null=True)
+
     class Meta:
-        db_table = 'retention'
-    client_id = models.UUIDField()
-    study = models.CharField(max_length=255)
+        db_table = 'studies'
```

Comment:
The `db_table` name here means we get one table named `studies` and another named `api_state`.